Gemfile

```diff
@@ -2,8 +2,3 @@ source "https://rubygems.org"
 
 gemspec
 
-gem 'mocha'
```

Comment:
This is a testing/stubbing gem. I think it is a `development_dependency` and not a `dependency`: https://github.com/freerange/mocha

Since it's not part of the `gemspec` today, that means that people can install `delsor` without installing `mocha`. Therefore try moving it to a `development_dependency` and see if tests still pass. If so, that's where it should be instead of `dependency`

Reply:
Done. Now `mocha` is in development_dependency in gemspec. tests pass without problem.